openmdao/drivers/tests/test_genetic_algorithm_driver.py

```diff
@@ -353,6 +353,26 @@ def test_SimpleGADriver_missing_objective(self):
 
         self.assertEqual(exception.args[0], msg)
 
+    def test_vectorized_constraints(self):
+        prob = om.Problem()
+        model = prob.model
+
+        dim = 2
+        model.add_subsystem('x', om.IndepVarComp('x', np.ones(dim)), promotes=['*'])
+        model.add_subsystem('f_x', om.ExecComp('f_x = sum(x * x)', x=np.ones(dim), f_x=1.0), promotes=['*'])
+        model.add_subsystem('g_x', om.ExecComp('g_x = 1 - x', x=np.ones(dim), g_x=np.zeros(dim)), promotes=['*'])
+
+        prob.driver = om.SimpleGADriver()
+
+        prob.model.add_design_var('x', lower=-10, upper=10)
+        prob.model.add_objective('f_x')
+
+        # Without bugfix to genetic_algorithm_driver, using vectorized constraint limits causes an exception
```

Comment:
I would remove the comment, it is very specific for the PR. Once merged, it's meaning won't be clear for the users.

Reply:
Addressed by d45b1fb3bdfd441c28e73da613f8fd240f6b44d3